src/models/UserModel.js

```diff
@@ -2,15 +2,20 @@ import { dbConnection } from "../config/database.js";
 import logger from "../utils/logger.js";
 
 export const createUser = async (firstname, lastname, email, password) => {
+	const hashedPassword = await bcrypt.hash(password, 10);
+	req.hashedPassword = hashedPassword;
```

Comment:
remove this as well. If you run this, you should have an error because `req` is not defined